packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -13,6 +13,7 @@ import { Localize } from '@deriv/translations';
 import classNames from 'classnames';
 
 import './traders-hub.scss';
+import WalletsBanner from 'Components/wallets-banner';
```

Comment:
```suggestion
import WalletsBanner from 'Components/wallets-banner';
import './traders-hub.scss';
```
There's actually an order for imports. Cant find the documentation though